genbranches

```diff
@@ -49,14 +49,20 @@ RUN yum -y install https://repo.grid.iu.edu/osg/%(series)s/osg-%(series)s-%(dver
                    yum-plugin-priorities && \
     yum -y install %(repo_arg)s \
                    osg-wn-client \
-                   osg-wn-client-glexec \
                    redhat-lsb-core
 
 # Install Singularity
 RUN yum -y install --enablerepo=osg-upcoming-development singularity && \
     yum clean all
 """
 
+DOCKERFILE_IN_GLEXEC = r"""
+
+#Install GLEXEC
+yum -y install %(repo_arg)s \
+               osg-wn-client-glexec
```

Comment:
1. It needs to be `RUN yum -y install ...` because "yum" itself is not a Dockerfile command.
2. You should do the `&& yum clean all` at the end again to clear the yum cache, which will keep the size of the container down.